Public/Src/Tools/Execution.Analyzer/Analyzers.Core/Xldb/XLGToDBAnalyzer.cs

```diff
@@ -644,6 +654,8 @@ private void IngestAllPips()
                             {
                                 AddToDirectoryConsumerMap(directoryArtifact, pipId);
                             }
+
+                            AddToDbStorageDictionary(DBStoredTypes.ProcessPip, xldbSpecificPip.ToByteArray().Length);

```

Comment:
> AddToDbStorageDictionary(DBStoredTypes.ProcessPip, xldbSpecificPip.ToByteArray().Length); [](start = 27, length = 90)

where do you actually call 'write to db' method?

Reply:
At the end of Analyze

---
In reply to: [321891291](https://github.com/microsoft/BuildXL/pull/836#discussion_r321891291) [](ancestors = 321891291)